articles/synapse-analytics/troubleshoot/troubleshoot-synapse-studio-powershell.md

```diff
@@ -22,29 +22,47 @@ Azure Synapse Studio (preview) depends on a set of Web API endpoints to work pro
 
 ## Troubleshooting steps
 
-Right-click on the following link, and select "Save target as":
+- Right-click on the following link, and select "Save target as":
 
-- [Test-AzureSynapse.ps1](https://go.microsoft.com/fwlink/?linkid=2119734)
+  [Test-AzureSynapse.ps1](https://go.microsoft.com/fwlink/?linkid=2119734)
 
-Alternatively, you may open the link directly, and save the opened script file. Don't save the address of the link above, as it may change in the future.
-
-In file explorer, right-click on the downloaded script file, and select "Run with PowerShell".
+- Alternatively, you may open the link directly, and save the opened script file. Don't save the address of the link above, as it may change in the future.
 
+- In file explorer, right-click on the downloaded script file, and select "Run with PowerShell" or navigate to the path of the PowerShell file within the PowerShell application using `cd <path to powershell script>` . Then run the command `.\Test-AzureSynapse.ps1`
 ![Run downloaded script file with PowerShell](media/troubleshooting-synapse-studio-powershell/run-with-powershell.png)
 
-When prompted, enter the Azure Synapse workspace name that is currently experiencing a problem or that you want to test for connectivity, and press enter.
+ - In case you are getting the below error,then run the command  `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
+
+```
+\Test-AzureSynapse.ps1 is not digitally signed. You cannot run
+this script on the current system. For more information about running scripts and setting execution policy, see
+about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
+At line:1 char:1
++ .\Test-AzureSynapse.ps1
++ ~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
+    + FullyQualifiedErrorId : UnauthorizedAccess 
+    
+  ```
+
+<img width="597" alt="image" src="https://user-images.githubusercontent.com/106385112/170699061-fe884720-b695-428f-9b5a-5eb9b3da8cf7.png">
```

Comment:
```suggestion
<!-- <img width="597" alt="image" src="https://user-images.githubusercontent.com/106385112/170699061-fe884720-b695-428f-9b5a-5eb9b3da8cf7.png"> -->
```
all images displayed in docs.microsoft.com must be in the repository. I'm commenting this out for now.